packages/design-system-react/src/components/button-primary/ButtonPrimary.test.tsx

```diff
@@ -153,7 +150,7 @@ describe('ButtonPrimary', () => {
     );
 
     expect(screen.getByText('Please wait...')).toBeInTheDocument();
-    expect(screen.queryByText('Submit')).not.toBeInTheDocument();
+    expect(screen.getByText('Submit')).toHaveClass('invisible');
```

Comment:
Updating test to check for invisible class instead of the children not being in the document